### ISSUE TITLE
list index out of range

### ISSUE THREAD

**Issue by yousefvand** [darealshinji/mkv-extractor-qt5#5]
Sometimes application stuck in a buggy state and upon opening it would get closed fast with an error indicating in file [MKVExtractorQt5.py](https://github.com/darealshinji/mkv-extractor-qt5/blob/master/MKVExtractorQt5.py#L595-L603) some **list index out of range**.  Commenting out two lines solved the problem temporarily but some configuration would not be saved by this change:

```
if Option == "OutputSameFolder" and Value and Configs.contains("InputFile"):
	# Recherche de l'option OutputSameFolder
	# x = self.ui.configuration_table.findItems("OutputSameFolder", Qt.MatchExactly)[0].row()

	## Il faut bloquer le signal pour éviter un cercle vicieux et modifier la valeur visuelle en directe
	self.ui.configuration_table.blockSignals(True)
	# self.ui.configuration_table.item(x, 1).setText(str(Configs.value("InputFolder")))

	# Envoie de la nouvelle variable à la fonction de gestion des dossiers qui mettra à jour la variable
	self.OutputFolder(Configs.value("InputFolder"))

	# Déblocage des signaux
	self.ui.configuration_table.blockSignals(False)
```

**Comment by darealshinji** [comment]
A new release came out, v5.5.6. Can you check if that one fixes the issue?
If it's not fixed you should consider contacting the original author, since this here is not the official main repository of the project: hizo@free.fr

**Comment by yousefvand** [comment]
This bug is in latest version (5.5.6). Unfortunately I couldn't find the official repo for this project so assumed there is no one. I will send an email to author. Thanks.

**Comment by darealshinji** [comment]
There's no VCS repo as far as I know, only the PPA releases on Launchpad.
By the way, if you're interrested I made my own mkvextract GUI in C++: https://github.com/darealshinji/mkvextract-gui

**Comment by yousefvand** [comment]
Now this is fixed in v5.5.7
